### ISSUE TITLE
broken flags tests re argo

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1526]
```R
✖ | 187 5     | Flags [2.0 s]
──────────────────────────────────────────────────────────────────────
test_flags.R:119: failure: handleFLags with the built-in argo dataset
is.na(argoNew[["salinity"]][13, 2]) isn't true.

test_flags.R:120: failure: handleFLags with the built-in argo dataset
is.na(argoNew[["salinity"]][53, 8]) isn't true.

test_flags.R:123: failure: handleFLags with the built-in argo dataset
!all(is.na(argoNew[["salinity"]][1 == argo[["salinityFlag"]]])) isn't true.

test_flags.R:129: failure: handleFLags with the built-in argo dataset
is.na(argoNew2[["temperature"]][10, 2]) isn't true.

test_flags.R:133: failure: handleFLags with the built-in argo dataset
sum(is.na(argoNew[["salinity"]])) not equal to 140.
1/1 mismatches
[1] 106 - 140 == -34
──────────────────────────────────────────────────────────────────────

```

**Comment by dankelley** [comment]
I have turned myself around, with this. I will commit what I have now (even though it is doing flags incorrectly) and will fix this and #1525 tomorrow sometime. In the meantime, below is a demo (for myself) that I am naming the flags incorrectly in `R/argo.R` (i.e. note the stray `QC` in the names).
```R
> str(section@data$station[[1]]@metadata$flags)
List of 7
 $ salinity      : num [1:5] 2 3 3 2 2
 $ salinityBottle: num [1:5] 2 2 2 2 2
 $ oxygen        : num [1:5] 2 2 2 2 2
 $ silicate      : num [1:5] 2 2 2 2 2
 $ nitrite       : num [1:5] 2 2 2 2 2
 $ NO2+NO3       : num [1:5] 2 2 2 2 2
 $ phosphate     : num [1:5] 2 2 2 2 2
> str(argo@metadata$flags)
List of 6
 $ pressureQC           : num [1:56, 1:223] 1 1 1 1 1 1 1 1 1 1 ...
 $ pressureAdjustedQC   : num [1:56, 1:223] 2 2 2 2 2 2 2 2 2 2 ...
 $ salinityQC           : num [1:56, 1:223] 1 1 1 1 1 1 1 1 1 1 ...
 $ salinityAdjustedQC   : num [1:56, 1:223] 2 2 2 2 2 2 2 2 2 2 ...
 $ temperatureQC        : num [1:56, 1:223] 1 1 1 1 1 1 1 1 1 1 ...
 $ temperatureAdjustedQC: num [1:56, 1:223] 2 2 2 2 2 2 2 2 2 2 ...
```

**Comment by dankelley** [comment]
I think this is fixed now. I am closing this, but if the pending rhub/travisCI builds reveal problems, I will reopen it.

commit abaa27c01d157235b900960bb53166780ec55bf4
Author: dankelley <kelley.dan@gmail.com>
Date:   Thu Apr 11 09:43:29 2019 -0300

    fix argo errors introduced in last 2 days
